<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary_Window</class>
 <widget class="QDialog" name="Summary_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Summary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Teaching</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="teaching_header">
         <property name="text">
          <string>Acuity Star Teaching Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_start_year_teaching">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_to_teaching">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_end_year_teaching">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_teach">
         <column>
          <property name="text">
           <string notr="true">&lt;Use the 'Load Data' button below to add a teaching CSV to the summary&gt;</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Presentations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="presentation_header">
         <property name="text">
          <string>Acuity Star Presentation Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_start_year_presentations">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_to_presentations">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_end_year_presentations">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_pres">
         <column>
          <property name="text">
           <string notr="true">&lt;Use the 'Load Data' button below to add a presentation CSV to the summary&gt;</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
      <zorder>treeWidget_pres</zorder>
      <zorder>presentation_header</zorder>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Grants</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="grant_header">
         <property name="text">
          <string>Acuity Star Grant Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_start_year_grants">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_to_grants">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_end_year_grants">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_grant">
         <column>
          <property name="text">
           <string notr="true">&lt;Use the 'Load Data' button below to add a grant/trail CSV to the summary&gt;</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Publications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="publication_header">
         <property name="text">
          <string>Acuity Publication Information</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_start_year_publications">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_to_publications">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_end_year_publications">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget_pub">
         <column>
          <property name="text">
           <string notr="true">&lt;Use the 'Load Data' button below to add a publication CSV to the summary&gt;</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
      <zorder>publication_header</zorder>
      <zorder>treeWidget_pub</zorder>
      <zorder></zorder>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Select Year Range   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fromDate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="toDate"/>
     </item>
     <item>
      <widget class="QPushButton" name="dateFilterButton">
       <property name="text">
        <string>Filter</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/Run_Images/Images/filter_icon.png</normaloff>:/Run_Images/Images/filter_icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_load_file">
       <property name="text">
        <string>Load File</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/Run_Images/Images/floppy_icon.png</normaloff>:/Run_Images/Images/floppy_icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_graph">
       <property name="text">
        <string>Graph</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/Run_Images/Images/pie_graph_icon.png</normaloff>:/Run_Images/Images/pie_graph_icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
